Lib/test/libregrtest/main.py

```diff
@@ -237,6 +238,16 @@ def find_tests(self, tests: TestList | None = None) -> tuple[TestTuple, TestList
         if self.randomize:
             random.shuffle(selected)
 
+        for priority_test in reversed(self.prioritize_tests):
+            try:
+                selected.remove(priority_test)
+            except ValueError:
+                print(f"warning: --prioritize={priority_test} used"
+                        f" but test not actually selected")
+                continue
+            else:
+                selected.insert(0, priority_test)
```

Comment:
Performance enthusiasts are crying

Reply:
This operation will be performed up to the number of tests listed in `--prioritize`. So like, three times tops. If somebody really attempted to order *everything* we've got right now, that's a list of fewer than 500 elements.